.vib/grafana/cypress/cypress.json

```diff
@@ -0,0 +1,17 @@
+{  
+    "baseUrl": "http://localhost/",
+    "env": {
+      "username": "test_admin",
+      "password": "ComplicatedPassword123!4",
+      "smtp_name": "test_smtp",
+      "smtp_host": "smtp",
+      "smtp_address": "test@smtp.com",
+      "smtp_user": "smtp_user",
+      "panel_title": "Test panel",
+      "panel_description": "Test panel description",
+      "dashboard_title": "Test dashboard",
```

Comment:
I'd like to know your thoughts on this. To me, these are test data (hence, fixtures) instead of environment variables. IMHO, environment variables should be reserved to values that may change due to external factors (e.g. an SMTP account) or whose values extremely make the test output change (e.g. the username we log in to perform the actions).

These data here (ass well as `test_email_address` and `invite_user_name`) do simply look like regular mockup data for tests that would perfectly fit in the concept of fixture.

Reply:
I see your point. I have changed this file to contain only the necessary environment data like `baseUrl`, `username`, `password`.
I agree that this data doesn't belong here, the `fixtures` folder would be better.